proposals/nnnn-video-streaming-state.md

```diff
@@ -0,0 +1,165 @@
+# Enhancing onHMIStatus with a New Parameter for Video Streaming State
+
+* Proposal: [SDL-NNNN](nnnn-video-streaming-state.md)
+* Author: [Zhimin Yang](https://github.com/smartdevicelink/yang1070)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / RPC]
+
+## Introduction
+
+This proposal adds a new parameter `videoStreamingState` to `onHMIStatus` notification so that the SDL can notify an app stop video streaming. This helps SDL handle the use cases of the coexistence of a navigation app and a projection app or multiple navigation/projection apps.
+
+## Motivation
+
+Currently, without Projection apps, a mobile navigation app can start video streaming in HMI level `FULL` and can continue streaming while in HMI level `LIMITED` in order to have a good user experience. The app must stop video streaming after receiving `onHMIStatus` with HMI level `NONE` or `BACKGROUND`. If the app does not stop streaming after certain time, SDL will send a `StopService` Control Frame to the app in protocol layer. Therefore, the HMI level implies when a navigation app shall stop video streaming or not. There is no dedicate RPC message or parameter from SDL to mobile app to say ‘stop video streaming’ in application layer. 
```

Comment:
Please update the third sentence to "If the app does not stop streaming after a certain amount of time, SDL Core will send..."  Please also update the last sentence to "There is no dedicated RPC message or parameter from SDL Core to ..."

Reply:
updated

Reply:
This doesn't look to be fully updated (missing "a certain amount of time" and "dedicated".  Can you please review and revise?

Reply:
sorry, missed it.